config.default.yaml

```diff
@@ -99,25 +107,27 @@ electricity:
     epsilon_vres: 0.02 # share of total renewable supply
     contingency: 0 # fixed capacity in MW
 
-  extendable_carriers:
-    Generator: []
-    StorageUnit: []
-    Store: [battery, H2]
-    Link: []
-
-  conventional_carriers: [coal, OCGT, CCGT, oil]
-
   max_hours:
     battery: 6
     H2: 168
 
-  powerplants_filter:
+  extendable_carriers:
+    Generator: [solar, onwind, offwind-ac, offwind-dc, OCGT]
+    StorageUnit: []  # battery, H2
+    Store: [battery, H2]
+    Link: []  # H2_pipeline
+
+  powerplants_filter: (DateOut >= 2022 or DateOut != DateOut)
```

Comment:
I feel this may be dangerous; have you tested its use?
There may be NaN or missing values. A validation should be used when using such numbers.
We may merge them and care about this issue later, but it must be verified.
Obviously, this applies to all configs

Reply:
Hey @davide-f ,
Tested now [H2_pipeline] which had to be renamed to [H2 pipeline]. Thanks! I could also find in n.links.carrier.unique() the carrier "H2 pipelines" proving that it works.
 :100: . Added also the pipeline stuff to the `config.test1.yaml`.
 
 Regarding `powerplants_filter: (DateOut >= 2022 or DateOut != DateOut)`. It works! Testing it, the query successfully removes powerplants that were decommissioned in e.g. 2016.